packages/client/test/Voter.spec.ts

```diff
@@ -13,7 +13,6 @@ import {
     ALICE,
     buildSimpleNodeApi,
     buildTestAuthenticatedSharedSate,
-    buildValidPolkadotAccountId,
```

Comment:
Is this function still in use? If not, it may be removed from `Utils.js`.

Reply:
Correct, it's not really useful any more. Removed.